areas/system.md

```diff
@@ -11,6 +11,8 @@
 
 ## Frameworks
 
+* Flower [[Web]](https://flower.dev/)
+  * Flower: A Friendly Federated Learning Framework [[Paper]](https://arxiv.org/abs/2007.14390)
```

Comment:
I guess you can also add it to the "## System Design" section.

Reply:
I added the paper to system design and left the webpage of flower at the framework area.